torch/ao/quantization/utils.py

```diff
@@ -384,3 +385,11 @@ def is_leaf_or_only_parametrized(module):
         return len(module._modules) == 1 and 'parametrizations' in module._modules
     else:
         return False
+
+def transfer_parametrizations_and_params(from_mod, to_mod):
```

Comment:
optional: this looks like a generic util unrelated to quantization, we could put it in parametrization code so others could use it as well